### ISSUE TITLE
Image missing a text alternative (IU Accessibility Mandate)

### ISSUE THREAD

**Issue by mdalmau** [IUBLibTech/newton_chymistry#139]
21 occurrences : HTML pages with images need an "alt" attribute: 
<img border="0" src="NWB splash.jpg" alt="Network Workbench logo">

--------------------------
IU has a mandate that all web sites published on IU servers must meet WCAG I and II Accessibility requirements. If the requirements are NOT met, sites will be taken down. To learn more about this initiative, visit the [Digital Strategy Initiative](https://ucm.iu.edu/digital-strategy/index.html) web site.

- [ ] Ask Bill who should be added to the SiteImprove tool

**Comment by mdalmau** [comment]
@tubesoft : At the Newton meeting today, Bill said you would be working on these accessibility issues. I requested SiteImprove access for you. Once you have access, you will login using your @iu credentials. If you want to read more about SIteImprove and accessibility, LOGIN to the KB and [start here](https://kb.iu.edu/d/webs). 

Could you have a look at this issue and confirm it's something you can fix? Once you confirm, we can figure out a testing workflow.

**Comment by tubesoft** [comment]
@mdalmau Some of the occurrences are LSA pages. Should we pass those to @wehooper ?
So far, I am going to work on non-LSA occurrences.

**Comment by mdalmau** [comment]
@tubesoft : I am slightly worried that we will create a conflict when we merge since I am assuming a lot of the fixes you made to the rendition issues in that pending pull request are in the same files for the accessibility pull request. Can you undo commits until we get the first pull request merged and dev/production updated?

**Comment by tubesoft** [comment]
@mdalmau I committed to a different branch from rendition issues, so merging the pending request does not affect changes for  the accessibility.

**Comment by mdalmau** [comment]
@tubesoft I am not sure how to handle this in GitHub since you fixed the non-LSA pages. Maybe I should have made it into two issues? I won't close this one for now.

**Comment by mdalmau** [comment]
@tubesoft ... Trying to check accessibility compliance for Newton. @randalldfloyd did a production update yesterday, but I think this is still showing as an issue. I am generally unclear if we ever created a pull request for these accessibility issues. In looking over PR history, I don't think so.
